src/Model/Snippet.php

```diff
@@ -66,12 +66,19 @@ public function getSnippetSummary()
     {
         $provider = $this->getSnippetProvider();
         if ($provider) {
-            return $provider->getSummary((array)json_decode($this->SnippetParams, true));
+            return $provider->getSummary((array) json_decode($this->SnippetParams, true));
```

Comment:
Does this change meet the needs of some style guide?

Reply:
Well, that's not really a requirement, but rather how they do it in the [PSR-12 examples](https://www.php-fig.org/psr/psr-12/#61-unary-operators).
I didn't intend to include this change on its own, but it was rather a part of some other refactoring (I wanted to move this block into a separate private method), but then decided not to include that into this patch, and reverted back, but not completely - forgot to remove this space :)